c2rust-analyze/src/context.rs

```diff
@@ -84,11 +85,25 @@ pub struct AnalysisCtxt<'a, 'tcx> {
 
     pub local_decls: &'a LocalDecls<'tcx>,
     pub local_tys: IndexVec<Local, LTy<'tcx>>,
+    pub c_void_ptrs: IndexSet<Place<'tcx>>,
```

Comment:
Can this just be a `HashSet`?  Order doesn't seem important as only `.insert` and `.contains` are used.

Reply:
Why not put `c_void_ptrs` right next to `special_casts`?  There might be a reason I'm missing, but otherwise, it'd make sense to put them together and make it a bit easier to understand they're closely related.

Reply:
875aad88